package.json

```diff
@@ -10,14 +10,17 @@
     "test": "standard && tap test.js"
   },
   "author": "David Mark Clements",
+  "contributors": ["Johannes Fahrenkrug"],
   "license": "MIT",
   "devDependencies": {
-    "standard": "^7.1.2",
-    "tap": "^6.2.0"
+    "standard": "^12.0.1",
+    "tap": "^13.1.8"
```

Comment:
@davidmarkclements this increase the minimum version of this module to Node 8. Are you ok with this?

Reply:
yeah but we'll have to release a major

Reply:
I'm -1 for that. I'll drop Node 6 when pino drops Node 6.

Reply:
I've reverted tap back to version 12.x which re-introduces support for Node 6.